_mobile/android.md

```diff
@@ -71,9 +71,9 @@ Further in this doc you can find how to rebuild it only for specific list of and
 
 `org.pytorch:pytorch_android_torchvision` - additional library with utility functions for converting `android.media.Image` and `android.graphics.Bitmap` to tensors.
 
-#### 4. Reading static image from android asset
+#### 4. Reading image from android asset
 
-All logic happens in [org.pytorch.helloworld.MainActivity](https://github.com/pytorch/android-demo-app/blob/master/HelloWorldApp/app/src/main/java/org/pytorch/helloworld/MainActivity.java#L31-L69).
+All logic happens in [`org.pytorch.helloworld.MainActivity`](https://github.com/pytorch/android-demo-app/blob/master/HelloWorldApp/app/src/main/java/org/pytorch/helloworld/MainActivity.java#L31-L69).
 As a first step we read `image.jpg` to `android.graphics.Bitmap` using standard android api. 
```

Comment:
"As a first step we read `image.jpg` to `android.graphics.Bitmap` using [the] standard [A]ndroid [API]."